src/types/models/MatrixModel.jl

```diff
@@ -21,7 +21,7 @@ mutable struct MatrixModel <: AbstractMetabolicModel
     xu::Vector{Float64}
     rxns::Vector{String}
     mets::Vector{String}
-    grrs::Vector{Maybe{GeneAssociation}}
+    grrs::Vector{Maybe{Vector{Vector{String}}}}
```

Comment:
Why doesn't this use the isozmes? (in the other case use GeneAssocDNF or a similar type name, this is bad)

Reply:
So isozymes is an ObjectModel only type of struct. Will add something like GeneAssocDNF  to represent those vector vector strings